### PR TITLE
move credentials as separate package

### PR DESCRIPTION
**Pull request by balamurugana**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: cmd/lock-instrument.go [hunk 1]
```diff
@@ -258,13 +257,5 @@ func (n *nsLockMap) deleteLockInfoEntryForOps(param nsParam, opsID string) error
...
-	const opsIDLen = 16
-	opsIDBytes := make([]byte, opsIDLen)
-	if _, err := rand.Read(opsIDBytes); err != nil {
-		panic(err)
-	}
-	for i := 0; i < opsIDLen; i++ {
-		opsIDBytes[i] = alphaNumericTable[opsIDBytes[i]%alphaNumericTableLen]
-	}
-	return string(opsIDBytes)
+	return mustGetUUID()
```
Comment (donatello) [inline cmd/lock-instrument.go:260]: This seems to be unrelated to moving credentials as a separate package - could we note it in the commit message?

**Review by donatello — APPROVED** [review]
LGTM - there seems to be a unrelated change though that we should note in the commit message.

File: cmd/lock-instrument.go (hunk 1 above)
Comment (balamurugana) [reply cmd/lock-instrument.go:260]: No. Previously its using `alphaNumericTable` from credential.go.  As its moved to separate package in this PR, its required to do this change as well.

**Review by harshavardhana — APPROVED** [review]
